common/acknowledgements.html

```diff
@@ -1,7 +1,7 @@
 <section class="appendix informative section" id="acknowledgements">
 	<h3>Acknowledgments</h3>
 	<p>The following people contributed to the development of this document.</p>
-	<div data-include="acknowledgements/aria-wg-active.html" data-include-replace="true"/>
-	<div data-include="acknowledgements/aria-contributors.html" data-include-replace="true"/>
-	<div data-include="acknowledgements/funders.html" data-include-replace="true"/>
+	<div data-include="acknowledgements/aria-wg-active.html" data-include-replace="true"></div>
```

Comment:
Can you remove this from this PR. The common directory is maintained in the w3c/aria-common repository.

Reply:
done